modules/runner/src/integrationTest/java/org/apache/ignite/internal/calcite/ItMixedQueriesTest.java

```diff
@@ -98,10 +97,10 @@ static void initTestData() {
     }
 
     /** Tests varchar min\max aggregates. */
-    @Disabled("https://issues.apache.org/jira/browse/IGNITE-15107")
     @Test
     public void testVarCharMinMax() {
-        sql("CREATE TABLE TEST(val VARCHAR primary key, val1 integer);");
+        // replicas and partitions params are affected by https://issues.apache.org/jira/browse/IGNITE-15655 issue, default is not working for now.
```

Comment:
mentioned issue about random timeouts, not about general performance

Reply:
mentioned test ok in TC and locally, thus i unmute it.